CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 ## 0.17.11-dev
+  * Add support for Flutter locale split.
+
+## 0.17.10+2
+  * Allow null safe code when parsing.
+
+## 0.17.10+1
   * Generate code that passes analysis with `implicit-casts: false`.
+  * Allow use of `MessageExtraction` and `MessageGeneration` without `File`.
```

Comment:
Might want to check the publish package to make sure these changes were actually in that release.

Reply:
Seems like the `arb` bits DID NOT get in that release, seeing as the code is here. But I'm not sure given the borked history